changelogs/fragments/7330-redfish-utils-oem-params.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_utils module utils - add support for ``Oem parameters`` in the ``MultipartHTTPPushUpdate`` command in ``redfish_command`` feature (https://github.com/ansible-collections/community.general/issues/7331).
```

I would formulate this as a new feature for redfish_command - that's how users are going to use this:
```suggestion
  - redfish_command - add new option ``update_oem_params`` for the ``MultipartHTTPPushUpdate`` command (https://github.com/ansible-collections/community.general/issues/7331).
```

---

Good call - appreciate for the feedback!